Example/Tests/CryptoTests.swift

```diff
@@ -0,0 +1,57 @@
+ //
+//  CryptoTests.swift
+//  E3db_Tests
+//
+
+import XCTest
+@testable import E3db
+
+class CryptoTests: XCTestCase, TestUtils {
+
+    func testFileEncryptDecryptEqual() {
+        guard let srcUrl = FileManager.tempBinFile(),
+              let dstUrl = FileManager.tempBinFile() else {
+            return XCTFail("Could not open files")
+        }
+        defer {
+            try! FileManager.default.removeItem(at: srcUrl)
+            try! FileManager.default.removeItem(at: dstUrl)
+        }
+
+        do {
+            let data = Data(repeatElement("testing", count: 10).joined().utf8)
+            try data.write(to: srcUrl)
+            let accessKey = Crypto.generateAccessKey()!
+            let encrypted = try Crypto.encrypt(fileAt: srcUrl, ak: accessKey)
+            defer {
+                try! FileManager.default.removeItem(at: encrypted.url)
+            }
+            try Crypto.decrypt(fileAt: encrypted.url, to: dstUrl, ak: accessKey)
+            let input  = try FileHandle(forReadingFrom: dstUrl)
+            let buffer = input.readDataToEndOfFile()
+            input.closeFile()
+            XCTAssertEqual(buffer, data)
+        } catch {
+            return XCTFail(error.localizedDescription)
+        }
+    }
+
+    func testMD5() {
```

Comment:
Is this test looking at the right thing? I looks like this is the only call to Crypto.md5

Reply:
Yep, just testing that the MD5 functionality is working as expected.